projects/eval-adxl355-pmdz/Makefile

```diff
@@ -1,7 +1,7 @@
 # Select the example you want to enable by chossing y for enabling and n for disabling
-DUMMY_EXAMPLE = n
+DUMMY_EXAMPLE = y
```

Comment:
these flags should be modified in the Makefile.

Reply:
they shouln't be added in another file. you should call from the command line with these flags. similar to what is done in builds.json

Reply:
okay, than will just discard the changes